deploy/handler/operator.yaml

```diff
@@ -204,6 +212,10 @@ spec:
           imagePullPolicy: {{ .HandlerPullPolicy }}
           command:
             - manager
+          resources:
+            requests:
+              cpu: "50m"
```

Comment:
maybe worth to be checked as well ?
atm k8s nmstate is pretty cookie monster

```
plain start |  33m / 67Mi
create / delete 100 bridges |   821m / 108Mi
Steady state after it |   38m / 86Mi
```

Reply:
For the runtime during configuration, I would keep this lower than 0.8 CPU for sure. We know it is very hungry, but I don't think we can justify reserving that much resources for spikes during the configuration.

It would be interesting to see if knmstate that is limited for 50m is still able to perform the whole configuration (although slower) or not

Reply:
50m is not good enough (Processing for unreasonable time / forever), looking to find a better number

Reply:
how many nodes we prefer to be on the cluster?

note:
1 node, latest CNAO, 1 bridge on non primary interface - takes 110 seconds to be created
without setting any limits

Reply:
Since the effort to define limits is mostly driven by single node usecases, we have it simple here.

Reply:
50m works but 3 times slower, and it might also not be safe, updated this with some other changes, tested them all together on CNAO